taipy/core/cycle/cycle.py

```diff
@@ -28,6 +28,30 @@
 class Cycle(_Entity, _Labeled):
     """An iteration of a recurrent work pattern.
 
+    Data applications are often used to solve business problems that operate periodically
```

Comment:
The thing that "operate periodically" from the sentence can be the "business problems", which is not correct.

```suggestion
    Business problems often require data applications that operate periodically
```

Reply:
You are right. the sentence is not clear. Please have a look to my other comment containing an alternative proposal.